demo/index.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+    <script type="module">
+      import '@polymer/polymer/lib/elements/dom-bind.js';
+      import '@polymer/iron-ajax/iron-ajax.js';
+      import '../default-theme.js';
+      import '../iron-doc-viewer.js';
+    </script>
+
+    <custom-style>
+      <style is="custom-style" include="iron-doc-default-theme"></style>
+    </custom-style>
```

Comment:
Does this work? From what I tried with `include="demo-pages-shared-styles"` in other elements, this needs to be stamped out in a `<script type="module">`. I made a suggestion regarding this in the doc.

Reply:
Yes, it works for me as-is.